### ISSUE TITLE
Implement Airflow Amplitude Operator task

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/data-infra#1126]
After #1046 is merged into https://github.com/cal-itp/data-infra/pull/1116, this task will continue to complete https://github.com/cal-itp/data-infra/issues/960. In this ticket, Amplitude event data in a Pandas dataframe will be saved into GCS.

## Airflow Operator task

4. [x] In the class `__init__()`, take in any API keys etc. and remember to call `super().__init__(**kwargs)`
5. [x] In the class implement an `execute(self, context)` method that calls the `amplitude_to_df` function and saves the resulting DataFrame to the GCS as a JSON file using `calitp.save_to_gcfs()`
```
 save_to_gcfs(df.to_csv(index=False).encode(), f"{gcs_file}", use_pipe=True)
```
6. [x] Import the new operator class in [`airflow/plugins/operators/__init__.py`](https://github.com/cal-itp/data-infra/blob/main/airflow/plugins/operators/__init__.py)
7. [x] In `airflow/dags/amplitude_loader` create a [`gusty`](https://github.com/chriscardillo/gusty) YAML file defining a DAG that uses `AmplitudeToFlattenedJSONOperator` to create and upload a JSON file to GCS ~and another that creates the external table in BigQuery~ (split out the latter as #1186). See the DAGs under [`rt_loader/`](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/rt_loader) as an example.

## Task details
- Job should run every day, a full California day
- Start and End: Start (8AM UTC) and End (7:59PM UTC) times should reflect 1 day
- Start: `20220201T00`
- End: `20220201T23`

Note: `The specified date range is the time of when the event data was uploaded to Amplitude servers (see server_upload_time) not when the event actually happened (client_time). The Export API returns timestamp the values in UTC.` https://developers.amplitude.com/docs/export-api#status-codes Data analysts can use `client_event_time ` to categorize data by when the event actually happened.

**Comment by thekaveman** [comment]
👍 Great write-up. Couple of things to keep in mind:

* We want to have a solid file naming convention to get useful partitioning and organization in GCS. @atvaccaro has posted at length about the importance of this in Slack! I think we talked about prefixing by project, then naming by date to the day level? Something like `{bucket-name}/amplitude/{project}/{yyyy-mm-dd}.jsonl` - does that make sense?

* The `gusty` files (naming etc.) should be specific to the project (in this case, `benefits`).

**Comment by angela-tran** [comment]
#1200 was merged
